src/sidebar/helpers/theme.js

```diff
@@ -62,7 +62,7 @@ export function applyTheme(themeProperties, settings) {
 
   themeProperties.forEach(themeProp => {
     const propertyName = supportedThemeProperties[themeProp];
-    const propertyValue = settings.branding[themeProp];
+    const propertyValue = settings.branding?.[themeProp];
```

Comment:
The `if` statement above checks that is `setting.branding` is defined. The compiler unfortunately still complains. I decided to solved in this way (I could have ignore the error, or cast)

Reply:
The way I usually resolve this is by assigning to the property in the parent scope (eg. `const branding = settings.branding`) and then referencing that in the callback. This makes it easier for TS to see that the value can't change. In this context you could also just change the `forEach` to a `for ... of` loop.